### DISCUSSION TITLE
Not Building Apps

### DISCUSSION THREAD

**Discussion by sorydima** [codemagic-ci-cd/codemagic-docs#2786] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Builds numbers are:

670fb45c3c9782e14b7775d2
670fb1d0451392dc76728bd5
6702aa2ca3035034e55df6c0
6702a8f98b813c75037be2df

### Expected results

Build done for each one. 

### Actual results

errors ..

### Build id (optional)

670fb45c3c9782e14b7775d2 670fb1d0451392dc76728bd5 6702aa2ca3035034e55df6c0 6702a8f98b813c75037be2df

**Comment by icarusdust** [comment]
Hi, as I see you have multiple threads open on the same issue in different channels. I will go ahead and close it here, so you can continue in other threads.
